osu.Game.Tests/Visual/Multiplayer/TestSceneGameplayChatDisplay.cs

```diff
@@ -126,11 +126,11 @@ public void TestFocusToggleViaAction()
             AddStep("set not expanded", () => chatDisplay.Expanded.Value = false);
             AddUntilStep("is not visible", () => !chatDisplay.IsPresent);
 
-            AddStep("press tab", () => InputManager.Key(Key.Tab));
+            AddStep("press enter", () => InputManager.Key(Key.Enter));
             assertChatFocused(true);
             AddUntilStep("is visible", () => chatDisplay.IsPresent);
 
-            AddStep("press tab", () => InputManager.Key(Key.Tab));
```

Comment:
Just noting that `TestFocusOnTabKeyWhenNotExpanded()` is now the same as `TestFocusToggleViaAction()`. Maybe remove one?